scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/OfferStrategy.java

```diff
@@ -39,7 +39,10 @@ protected boolean isAtLeastOneESNodeRunning() {
                 // TODO (PNW): Better get HTTP address method. We do this everywhere.
                 InetSocketAddress clientAddress = taskList.get(clusterState.getTaskList().get(0).getTaskId().getValue()).getClientAddress();
                 String hostAddress = NetworkUtils.addressToString(clientAddress, configuration.getIsUseIpAddress());
-                return Unirest.get(hostAddress).asString().getStatus() == 200;
+                final int hostHttpStatus = Unirest.get(hostAddress).asString().getStatus();
+                // If the quorum requirement is set higher than the number of nodes running,
+                // we will get back a 503.
+                return hostHttpStatus == 200 || hostHttpStatus == 503;
```

Comment:
Is it worth checking the body for a specific error, to ensure that is not another error.  IE check body for `Not enough active copies to meet write consistency of [QUORUM] `